app/configuration/Config.scala

```diff
@@ -128,8 +128,7 @@ object Config {
   val getAddressIOApiUrl = config.getString("get-address-io-api.url")
   val getAddressIOApiKey = config.getString("get-address-io-api.key")
 
-  val fulfilmentLookupApiKey = config.getString("fulfilment-lookup-api.key")
-
-  val fulfilmentLookupApiUrl = config.getString("fulfilment-lookup-api.url")
+  def fulfilmentLookupApiKey(env: String) = config.getString(s"fulfilment-lookup-api.$env.key")
+  def fulfilmentLookupApiUrl(env: String) = config.getString(s"fulfilment-lookup-api.$env.url")
```

Comment:
there's already a stageConfig for zuora and salesforce -
 https://github.com/guardian/subscriptions-frontend/pull/985/files#diff-f2117ad9c5563c42557d76930e24b9adL69
https://github.com/guardian/subscriptions-frontend/pull/985/files#diff-f2117ad9c5563c42557d76930e24b9adL85
does this fit in with the same structure in the config?

Reply:
Yeah, I considered this for a while too - in the end I decided not to bloat TouchPoint backend any further. I think this whole area needs to be tidied up / renamed, but I'll save that one for a rainy day...